labgrid/remote/client.py

```diff
@@ -1181,6 +1203,11 @@ def main():
     tmc_subparser.add_argument('action', choices=['info', 'values'])
     tmc_subparser.set_defaults(func=ClientSession.tmc_channel)
 
+    subparser = subparsers.add_parser('write-image', help="Write an image onto mass storage")
```

Comment:
The majority of help texts start lower case.

Reply:
Fixed.